### DISCUSSION TITLE
Strimzi Kafka ReadWriteMany accessmode

### DISCUSSION THREAD

**Discussion by CoMaDaniel** [strimzi/strimzi-kafka-operator#10304] [Q&A] [answered]
Hello all.

Hope this is not a duplicate.

I am trying to deploy Strimzi with Kafka, but one of the requirements is to use ReadWriteMany accessmode for the PVC.
I cannot find a good example, nor could I find the accessMode option in the Kafka CRD.

Can anyone help with that?
Thank you

**Comment by scholzj** [comment, marked as answer]
Kafka does not use ReadWriteMany volumes. So while you should be able to use them through the PVCs, they will be used as read write once only. Also, please keep in mind that Kafka needs block storage. The typical read write many volumes such as NFS are usually not compatible with Kafka.

**Reply by CoMaDaniel** [reply]
Duly noted. I understand.

Thank you very much for your prompt response.

Have a great day!
